src/app/beer_garden/api/http/__init__.py

```diff
@@ -370,6 +371,6 @@ def _event_callback(event):
         beer_garden.requests.handle_event,
     ]:
         try:
-            handler(event)
+            handler(deepcopy(event))
```

Comment:
There's a built-in assumption here that I feel should be examined.

Without getting into the specifics of how any one handler does its business, the code here assumes that if a handler throws an exception, the repr of the *original* version of the event would make for a meaningful error message to the user. In other words, it assumes that the handler doesn't mutate the event in a way that I'll care about (and will be able to see) before it blows up.

Perhaps in our specific case this won't matter at all, but as a general pattern I can see this being problematic.

Reply:
It's intuition that handlers should be writing information on errors into the *exception* they throw back here, not the event, right? 

Seeing the original event seems like enough information for a normal user. Showing how an event handler modified an event in an error message seems like information only a dev would need to know while debugging.